src/v2/controllers/FormController.ts

```diff
@@ -349,6 +349,8 @@ export default Controller.extend({
       errorObj = { responseJSON: { errorSummary: loc('error.unsupported.response', 'login')}};
     }
 
+    this.options.appState.set('messages', idxStateError.messages)
```

Comment:
I need access to the original messages object to retrieve some custom data. See mock response [here](https://github.com/okta/okta-signin-widget/pull/2597/commits/b5deef5bd25891189024c6ba855db92499daea60#diff-cf39648fe1b124ef6de42e035db9fbe3d107450fd14c082441ad0100e2c021ccR12).

Reply:
Alternatively, could pass `messages` to `showCustomFormErrorCallout` on [line 355](https://github.com/okta/okta-signin-widget/pull/2597/commits/b5deef5bd25891189024c6ba855db92499daea60#diff-72a0e868d27f4dfd26d36bcba1d860aa25999401fa00788f3ac694304043dd4dR355)